test/integration/helpers.js

```diff
@@ -1110,6 +1111,21 @@ function createSapiClient(callback) {
 
 
 /**
+ * Get a simple restify JSON client to PAPI.
+ */
+function createPapiClient(callback) {
+    createClientOpts('papi', function (err, opts) {
```

Comment:
Yep, good catch! Unfortunately this file is missing a lot of asserts. I'll have to add those in too.